config/default/user.role.authenticated.yml

```diff
@@ -22,6 +22,7 @@ permissions:
   - 'access content'
   - 'access fontawesome additional settings'
   - 'access shortcuts'
+  - 'show shortcut menu in the toolbar'
```

Comment:
Move this permission to proper roles. Authenticated users should have even less access than they currently have.

Reply:
Removed the permission, it was not being used because that functionality was reimplemented in another way.